radars/IL.csv

```diff
@@ -68,7 +68,7 @@ When decision will be revisited: 01/01/2022</p>
 <p>When decision made: 5/1/2020</p>
 <p>When decision will be revisited: 5/1/2021</p>
 "
-Redux,Frontend,Invest,confident,"<p><a href=""https://redux.js.org/"">Redux</a> is a predictable state container for JavaScript applications.</p>
+Redux,Frontend,Invest,lock,"<p><a href=""https://redux.js.org/"">Redux</a> is a predictable state container for JavaScript applications.</p>
```

Comment:
Can we update decision made and decision revisited details since we are updating this? (this comment probably applies to all of the things we updated in this pr)

Reply:
I agree and made the changes.